docs/Developers/Review.rst

```diff
@@ -117,6 +117,15 @@ For the Reviewers
    behaviour well. E.g. for bear tests just testing for a good and bad file
    is **not** sufficient.
 
+.. note::
+
+    While reviewing Pull request/``Patch`` for ``difficulty/low`` issue make
+    sure the Patch solves the issue and doesn't create any further issues.
+
+    You need to understand the function of the code and check whether it
+    is optimal or not. We need manual review to find the problems which can't
+    be found automatically since the tests passes are automated checks.
```

Comment:
"since the tests passes are automated checks" - doesn't seem to fit well 😉 You can omit this if you feel so....

Reply:
I think It defining more defining the things

Reply:
Actually, tests are already introduced and described in "Automated Review Process" and "For the Reviewers" sections above :)
Also, is there any specific reason of adding this as a "Note" rather continuing the same "For the Reviewers" list?